### ISSUE TITLE
Consider using another, more permissive license

### ISSUE THREAD

**Issue by Arkanosis** [0x010C/LiveRC2#1]
GPLv2 is not compatible with CC-BY-SA 3.0, which is the license used on Wikipedia.

(Irrelevant here, but for information: CC-BY-SA 4.0 is one-way compatible with GPLv3, which means CC-BY-SA 4.0-licensed work can be integrated into GPLv3-licensed work, but the opposite is not true.)

I suggest using a liberal license instead, such as MIT, BSD 2 clauses or ISC, which would:
 - make it compatible with CC-BY-SA 3.0, hence making sure there's no license hazard when syncing the repo to Wikipedia;
 - make it compatible with CC-BY-SA 3.0, allowing reuse of code within other scripts / gadgets ;
 - make it compatible with MIT/X11, hence allowing reuse of code within LiveRC 1.x.

In any case:
 - it would not allow to reuse code from Huggle (GPLv3, but it's not JS anyway) or other scripts / gadgets (CC-BY-SA 3.0);
 - it would allow reuse of code from LiveRC 1.x (MIT/X11).

**Comment by 0x010C** [comment]
Yes sure. I picked the GPLv2 because it's on of the most common license for MediaWiki Extensions, but the MIT is totally fine for me.

I've invited you as collaborator to the repo, so feel free to change the license :)

**Comment by Arkanosis** [comment]
Ok thanks! :)
